noseprogressive/wrapping.py

```diff
@@ -29,7 +29,11 @@ def unwrapping_raw_input(*args, **kwargs):
         sys.stdout = wrapped_stdout
         return ret
 
-    orig_raw_input = raw_input
+    try:
+        orig_raw_input = raw_input
+    except:
```

Comment:
If you change this to `except NameError` (or whatever actually gets thrown), I think we can merge this. Thanks for having a look!